index.html

```diff
@@ -24,257 +24,305 @@
-            <img src="images/icon-skills.svg" alt="skills-icon" class="about__card__img">
-          </div>
-
-          <div class="card__details">
-
-            <h3 class="card__text card__text--about">Skills</h3>
+        <div class="about__btn">
+          <a href="#" class="btn btn--green">get me resume</a>
+        </div>
 
-            <ul class="about__card__list">
-              <li>Github</li>
-              <li>Git</li>
-              <li>Terminal</li>
-              <li>Figma</li>
-              <li>Design</li>
-            </ul>
+        <ul class="card-container">
 
-          </div>
-        </li>
-      </ul>
-    </section>
+          <li class="about__card">
 
+            <div class="about__card__imgbox">
+              <img src="images/icon-languages.svg" alt="Languages-icon" class="about__card__img">
+            </div>
+
+            <div class="card__details">
+
+              <h3 class="card__text card__text--about">Languages</h3>
+
+              <ul class="about__card__list">
+                <li>javaScript</li>
+                <li>Ruby</li>
+                <li>Python</li>
+                <li>html</li>
+              </ul>
+
+            </div>
+          </li>
+
+          <li class="about__card">
+
+            <div class="about__card__imgbox">
+              <img src="images/icon-frameworks.svg" alt="frameworks-icon" class="about__card__img">
+            </div>
+
+            <div class="card__details">
+
+              <h3 class="card__text card__text--about">Frameworks</h3>
+
+              <ul class="about__card__list">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Angular</li>
+              </ul>
+
+            </div>
+          </li>
+
+          <li class="about__card">
+
+            <div class="about__card__imgbox">
+              <img src="images/icon-skills.svg" alt="skills-icon" class="about__card__img">
+            </div>
+
+            <div class="card__details">
+
+              <h3 class="card__text card__text--about">Skills</h3>
+
+              <ul class="about__card__list">
+                <li>Github</li>
+                <li>Git</li>
+                <li>Terminal</li>
+                <li>Figma</li>
+                <li>Design</li>
+              </ul>
+
+            </div>
+          </li>
+        </ul>
+      </section>
+    </main>
+    <footer class="contact">
+
+      <h3 class="title">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h3>
+
+      <form action="https://formspree.io/f/xzbopkkk" method="POST" id="my-form">
+
+        <input type="text" placeholder="Full name" maxlength="30" name="full-name" required>
+
+        <input type="email" placeholder="Email address" name="email" required>
+
+        <textarea placeholder="Enter text here" name="message" maxlength="500" required>  </textarea>
+
```

Comment:
Kindly implement these two changes for the `textarea`:

- [ ] An addition of placeholder with text **_Enter text here_**
- [ ] Ensure that the form can not submit without a user of your website filling the textarea form control. Although you used the `required` attribute for the `textarea`, it still submits to **_Formspree_** when I do not fill it. 🙏

Reply:
Hi @Gambit142 👋
Thank you for your review 🙏

But I think I have already did these two changes

- This is my code:

    <textarea placeholder="Enter text here" name="message" maxlength="500" required>  </textarea>

- I think I should delete the empty space between the opening and closing 'textarea' tag, Right?.

Reply:
Your `placeholder` text does not visible, that is the reason why I asked you to fix it. Kindly fix it